mne/io/reference.py

```diff
@@ -193,11 +197,22 @@ def add_reference_channels(inst, ref_channels, copy=True):
                      'unit': FIFF.FIFF_UNIT_V,
                      'coord_frame': FIFF.FIFFV_COORD_HEAD,
                      'loc': np.zeros(12)}
+        # see if "zeroth" EEG electrode present in dig points
+        dig_loc = [dl for dl in inst.info['dig'] if (
+                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
+                   dl['ident'] == 0)]
+        if len(dig_loc) > 1:  # is either 0 or 1, never >1 for Neuromag
+            warn('Multiple reference channel locations detected! Their '
+                 'location information will be ignored.')
```

Comment:
You misunderstood. I would like it if a warning was given if `len(`**`ref_channels`** `) > 1`. That is, if the user adds 2 or more reference channels. If there is a suitable `dig_loc` in that case (that shouldn't happen in a proper Neuromag file I guess, but theoretically...) than both reference channels will be assigned the same location. Consequently, any calls to topo/topomap functions will fail with the error "channel locations are not unique". Better in that case not to use location information at all.